hub/core/query/filter.py

```diff
@@ -57,12 +68,30 @@ def pg_filter_slice(pg_callback, indices: Sequence[int]):
     result: Sequence[List[int]]
     idx: List[List[int]] = [block.indices() for block in blocks]
 
+    query_id = str(uuid4())
+    dataset.send_query_progress(query_id=query_id, start=True, progress=0)
```

Comment:
Lets remove the `send_query_progress` calls from this PR but keep the method, and add them back in after dataset view saving is merged. (Alternatively you can also rebase this PR to that branch)